butterknife-compiler/src/main/java/butterknife/compiler/ButterKnifeProcessor.java

```diff
@@ -1215,11 +1230,11 @@ private static AnnotationMirror getMirror(Element element,
     return null;
   }
 
-  private Id getId(int id) {
-    if (symbols.get(id) == null) {
-      symbols.put(id, new Id(id));
+  private Id getId(QualifiedId qualifiedId) {
```

Comment:
QualifiedId already has the id as part of it right? why do you still need the symbol map